Sources/MapboxCoreNavigation/FreeDriveEventDetails.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import CoreLocation
+
+struct FreeDriveEventDetails: GlobalEventDetails {
```

Comment:
We should avoid the term “free drive” in client-side APIs, even if the server API is using that term. It’s confusing for a few reasons, not least of which is that it isn’t necessarily free. I realize this type isn’t public, but we may eventually need to add a related public type, so we would want to be consistent.

How does this type differ from PassiveNavigationEventDetails? It seems to be something that should be named very similarly.

Reply:
Agree, this second struct is unnecessary in free drive, we can reuse `PassiveNavigationEventDetails` type both for feedback and departure/arrival events, similar to `ActiveNavigationEventDetails` in active guidance.